install_scripts/fits.sh

```diff
@@ -7,13 +7,16 @@ if [ -f "$SHARED_DIR/install_scripts/config" ]; then
   . $SHARED_DIR/install_scripts/config
 fi
 
-DOWNLOAD_URL="http://projects.iq.harvard.edu/files/fits/files/fits-${FITS_VERSION}.zip"
-cd $DOWNLOAD_DIR
-if [ ! -f "fits.zip" ]; then
-  sudo curl $DOWNLOAD_URL > fits.zip
+if [ ! -d fits-$FITS_VERSION ]; then
+  DOWNLOAD_URL="http://projects.iq.harvard.edu/files/fits/files/fits-${FITS_VERSION}.zip"
+  cd $DOWNLOAD_DIR
+  if [ ! -f "fits.zip" ]; then
+    curl $DOWNLOAD_URL -o fits.zip
+  fi
+  unzip fits.zip
+  chmod a+x fits-$FITS_VERSION/*.sh
+  FITS_PATH="${DOWNLOAD_DIR}/fits-${FITS_VERSION}"
+  cd
+  echo "PATH=\${PATH}:$FITS_PATH" >> .bashrc
```

Comment:
Are you certain. Looks like it runs as ubuntu to me.  I reprovisioned and it works fine.  Are you having trouble?